.gitignore

```diff
@@ -28,3 +28,6 @@ breakpoints.txt
 
 # coverage output
 coverage.txt
+
+# go workspace
+go.work
```

Comment:
😲

Do you use the go work space stuff? I read the docs a few times, but could never fully get my head around the utility....

Reply:
yeah I used it cause vim kept complaining about the `cache` module, and my `go.work` is simply,

```
go 1.19

use ./cache
```

Meanwhile I don't understand the docs(they need to be improved tbh)😂, only trial and error got it working.